# Formulas for T-test

$$ \large z = \frac{\bar{x} - \mu}{\sigma} $$

Z: z-score 

x_bar: the mean of the sample or value of interest.

mu: The mean of the population, usually given. 
(we expect our sample to have about the same mean as the population if the null hypothesis is true)


sigma: the std of the population, usually also given.



If $p \lt \alpha$, we reject the null hypothesis.:

If $p \geq \alpha$, we fail to reject the null hypothesis.

In [3]:
z = (75-65)/2.7

In [4]:
z

3.7037037037037033

In the code above, the sample mean was 65 and the mean of the population was 75, and the sigma was 2.7. In a standarized format, the sample mean is 3.7 deviation away from the mean and sample mean only accounts .15% of the population mean.